modules/ROOT/examples/live-demos/inline-css/index.html

```diff
@@ -0,0 +1,52 @@
+<textarea id="inline-css" class="classic">
+  <h1>TinyMCE documents and Inline CSS documents</h1>
+
+  <h2>TinyMCE documents</h2>
+
+  <p><em>TinyMCE</em> documents consist of</p>
+
+  <ul>
+  <li>material marked up as HTML;</li>
+  <li>stylesheets defined by the <code>content_css</code> and <code>font_css</code> options; and</li>
+  <li>CSS defined by the <code>style_css</code> option.</li>
+  </ul>
+
+  <h2>Inline CSS</h2>
+
+  By contrast, <em>Inline CSS</em> documents consist of
+
+  <ul>
+  <li>a single file with all CSS styling applied inline to each HTML element.</li>
+  </ul>
+
+  <h2>The TinyMCE Inline CSS plugin</h2>
+
+  <p>The TinyMCE Inline CSS plugin takes, as input, a TinyMCE document, with its multiple components.</p>
```

Comment:
This is probably a bit confusing because the `inlinecss` plugin function doesn't take any inputs / arguments. It has `editor` in scope, so it's essentially "curried", but it doesn't take any arguments. So because sometimes "takes as input" is used to describe which arguments a function takes, this is a bit misleading.

Reply:
Text edited to remove ‘input’ and ‘output’. The Plugin now takes and returns.